packages/utils/src/shorthand.ts

```diff
@@ -0,0 +1,610 @@
+    'border-block-end-style',
+    'border-block-end-width',
+    'border-bottom-color',
+    'border-bottom-style',
+    'border-bottom-width',
+  ],
+  'border-block-start': [
+    'border-block-start-color',
+    'border-block-start-style',
+    'border-block-start-width',
+    'border-top-color',
+    'border-top-style',
+    'border-top-width',
+  ],
+  'border-bottom': [
+    'border-block-end-color',
+    'border-block-end-style',
+    'border-block-end-width',
+    'border-bottom-color',
+    'border-bottom-style',
+    'border-bottom-width',
+  ],
+  'border-color': [
+    'border-block-color',
+    'border-inline-color',
+    'border-block-start-color',
+    'border-block-end-color',
+    'border-inline-start-color',
+    'border-inline-end-color',
+    'border-top-color',
+    'border-right-color',
+    'border-bottom-color',
+    'border-left-color',
+  ],
+  'border-image': [
+    'border-image-outset',
+    'border-image-repeat',
+    'border-image-slice',
+    'border-image-source',
+    'border-image-width',
+  ],
+  'border-inline': [
+    'border-inline-start',
+    'border-inline-start-color',
+    'border-inline-start-style',
+    'border-inline-start-width',
+    'border-inline-end',
+    'border-inline-end-color',
+    'border-inline-end-style',
+    'border-inline-end-width',
+    'border-right-color',
+    'border-right-style',
+    'border-right-width',
+    'border-left-color',
+    'border-left-style',
+    'border-left-width',
+  ],
+  'border-inline-end': [
+    'border-inline-end-color',
+    'border-inline-end-style',
+    'border-inline-end-width',
+    'border-right-color',
+    'border-right-style',
+    'border-right-width',
+    'border-left-color',
+    'border-left-style',
+    'border-left-width',
+  ],
+  'border-inline-start': [
+    'border-inline-start-color',
+    'border-inline-start-style',
+    'border-inline-start-width',
+    'border-right-color',
+    'border-right-style',
+    'border-right-width',
+    'border-left-color',
+    'border-left-style',
+    'border-left-width',
+  ],
+  'border-left': [
+    'border-inline-start-color',
+    'border-inline-start-style',
+    'border-inline-start-width',
+    'border-inline-end-color',
+    'border-inline-end-style',
+    'border-inline-end-width',
+    'border-left-color',
+    'border-left-style',
+    'border-left-width',
+  ],
+  'border-radius': [
+    'border-start-start-radius',
+    'border-start-end-radius',
+    'border-end-start-radius',
+    'border-end-end-radius',
+    'border-top-right-radius',
+    'border-top-left-radius',
+    'border-bottom-right-radius',
+    'border-bottom-left-radius',
+  ],
+  'border-right': [
+    'border-inline-start-color',
+    'border-inline-start-style',
+    'border-inline-start-width',
+    'border-inline-end-color',
+    'border-inline-end-style',
+    'border-inline-end-width',
+    'border-right-color',
+    'border-right-style',
+    'border-right-width',
+  ],
+  'border-style': [
```

Comment:
Same as the other comment, if you fully agree:
```suggestion
  'border-style': [
    'border-top',
    'border-block',
    'border-inline',
    'border-left',
    'border-right',
    'border-bottom',
```